keyboards/bastardkb/skeletyl/keymaps/default/keymap.json

```
@@ -0,0 +1,26 @@
+{
+    "keyboard": "bastardkb/skeletyl",
```

```suggestion
    "keyboard": "bastardkb/skeletyl/promicro",
```
This must always be a valid keyboard name (regardless of whether it's the one currently being built) otherwise qmk_compiler complains. Might need to be fixed up in the previous refactorings.

---

Ah yep, hadn't realised that. I'll double check other PRs, ta.